bundle/Resources/config/default_settings.yml

```diff
@@ -0,0 +1,52 @@
+parameters:
+    # Defaults for Site API bundle semantic configuration
+    netgen_ez_platform_site_api.default.override_url_alias_view_action: false
+    netgen_ez_platform_site_api.default.use_always_available_fallback: true
+    netgen_ez_platform_site_api.default.fail_on_missing_fields: '%kernel.debug%'
+    netgen_ez_platform_site_api.default.render_missing_field_info: false
+
+    # Default view templates
+    netgen.ezplatform_site.default_view_templates.content.asset_image: 'NetgenEzPlatformSiteApiBundle:default:content/asset_image.html.twig'
```

Comment:
You should use Twig notation to reference the templates, bundle notation is deprecated. In this instance it would be `@@NetgenEzPlatformSiteApi/default/content/asset_image.html.twig`

(double `@` is intentional to prevent the template name to be parsed as a service)

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/130/commits/1423efe0bd75f76b261a34a38f7ad846a295ed38